### PR TITLE
Feature/not found page

### PR DESCRIPTION
**Pull request by pawelkloc**

### CODE REVIEW

**Review by YarynaPuhach — APPROVED** [review]

File: src/App.tsx
Comment (YarynaPuhach) [inline src/App.tsx:1]: why /components/NotFoundPage/NotFoundPage';
if we have index.ts, shouldnt it be possible to use simply (/components/NotFoundPage';)

File: src/components/NotFoundPage/page-not-found.png
Comment (TideTwo) [inline src/components/NotFoundPage/page-not-found.png:1]: we already have this image in src

**Review by TideTwo — CHANGES_REQUESTED** [review]
you need to add whatever folder ".history" is to your gitignore

**Review by TideTwo — CHANGES_REQUESTED** [review]
You are adding files, like the Cart item, on the branch "not found page" they are not related, please remove those changes from the branch

**Pull request closed without merging**